internal/testkeys/testkeys_test.go

```diff
@@ -65,6 +65,10 @@ func TestKeyCount(t *testing.T) {
 	}
 	testCases := map[params]int64{
 		{26, 1}: 26,
+		{26, 2}: 702,
+		{26, 3}: 18278,
+		{26, 4}: 475254,
+		{26, 5}: 12356630,
```

Comment:
I think this is just testing `keyCount` which uses `n + n^2 + ... + n^l` to calculate the size of the keyspace, rather than actually generating all of the keys themselves. This might still be overkill given we already have the other test cases; I included it mostly as a sanity check.

Reply:
Good point. Sounds good to me then.